pkgs/tools/security/imhex/default.nix

```
@@ -0,0 +1,89 @@
+{ lib
+, stdenv
+, cmake
+, fetchFromGitHub
+, mbedtls
+, gtk3
+, pkgconfig
+, capstone
+, libGLU
+, glfw3
+, file
+, python3
+, curl
+, xlibs
+}:
+
+stdenv.mkDerivation rec {
+  pname = "ImHex";
+  version = "1.11.1";
+
+  src = fetchFromGitHub {
+    fetchSubmodules = true;
+    owner = "WerWolv";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-3S6mQ/fmxT2m1oH/MRn+3egj+HPYzaZys82nZ3RcE1I=";
+  };
+
+  patterns_src = fetchFromGitHub {
+    owner = "WerWolv";
+    repo = "ImHex-Patterns";
+    rev = "47fc94928960397feba24dd687f9b1c803213b34";
+    sha256 = "sha256-bufoyXbqRS/hIOU03ACBfHdrATTnuW5R1IiNLF/B9t4=";
+  };
+
+  nativeBuildInputs = [ cmake python3 ];
+
+  buildInputs = [
+    mbedtls
+    gtk3
+    pkgconfig
+    capstone
+    libGLU
+    glfw3
+    file
+    curl.dev
+    xlibs.libXdmcp.dev
```

```suggestion
    curl
    xlibs.libXdmcp
```

Is that not choosen automatically?